spec/rails_helper.rb

```diff
@@ -62,4 +63,12 @@
 
   # serializer_spec_helper
   config.include SerializerSpecHelper, type: :serializer
+
+  # clean up carrierwave uploads
+  # https://github.com/carrierwaveuploader/carrierwave/wiki/How-to:-Cleanup-after-your-Rspec-tests
+  config.after(:all) do
+    if Rails.env.test?
+      FileUtils.rm_rf(Dir["#{Rails.root}/spec/support/uploads"])
+    end
+  end
```

Comment:
This is mostly just a nitpick, but it would be really nice if this could be in a helper of some sort so that others who use carrierwave can efficiently write specs.